akka-cluster-typed/src/test/scala/akka/cluster/ddata/typed/scaladsl/ReplicatorSpec.scala

```diff
@@ -72,8 +72,8 @@ object ReplicatorSpec {
               replicator ! Replicator.Get(Key, Replicator.ReadLocal, getResponseAdapter, Some(replyTo))
               Behaviors.same
 
-            case GetCachedValue(replyTo) ⇒
-              replicator ! Replicator.Get(Key, Replicator.ReadLocal, getResponseAdapter, Some(replyTo))
+            case GetCachedValue(replyTo: ActorRef[Int] @unchecked) ⇒
```

Comment:
`GetCachedValue` is a concrete type with no type parameter, so `replyTo` can never be anything else than `ActorRef[Int]` so there should be no need for a type in the pattern here.